crates/c-api/include/wasmtime/store.h

```diff
@@ -164,6 +164,23 @@ WASM_API_EXTERN void wasmtime_context_gc(wasmtime_context_t* context);
  */
 WASM_API_EXTERN wasmtime_error_t *wasmtime_context_add_fuel(wasmtime_context_t *store, uint64_t fuel);
 
+/**
+ * \brief Set fuel to this context's store for wasm to consume while executing.
+ *
+ * For this method to work fuel consumption must be enabled via
+ * #wasmtime_config_consume_fuel_set. By default a store starts with 0 fuel
+ * for wasm to execute with (meaning it will immediately trap).
+ * This function must be called for the store to have
+ * some fuel to allow WebAssembly to execute.
+ *
+ * Note that at this time when fuel is entirely consumed it will cause
+ * wasm to trap. More usages of fuel are planned for the future.
```

Comment:
Is this just copy-pasted? Because with the Rust API you can yield on out-of-fuel and I thought that the C API supported async as well now, but I am not sure if the fuel bits for that are connected or not.

Reply:
Ah yeah I see that it is copy-pasted. Mind removing this paragraph here and in the other place?

If you want to go above and beyond, you could replace it with a bit about how one might configure execution to trap or async yield on out of fuel via https://docs.rs/wasmtime/latest/wasmtime/struct.Store.html#method.out_of_fuel_async_yield and https://docs.rs/wasmtime/latest/wasmtime/struct.Store.html#method.out_of_fuel_trap

Reply:
Done! Thanks for catching :)